modular_skyrat/modules/modular_reagents/code/recipes/medicine_recipes.dm

```diff
@@ -19,3 +19,20 @@
 	if(off_cooldown(holder, equilibrium, 10, "lidocaine"))
 		explode_attack_chem(holder, equilibrium, /datum/reagent/inverse/lidocaine, 5)
 		explode_invert_smoke(holder, equilibrium)
+
+/datum/chemical_reaction/medicine/ataraxydone
```

Comment:
Have you tried making that chem yourself during testing? Just how complicated is it to make, and how much of a pain is that reaction to keep in the pure range?